spacy/attrs.pxd

```diff
@@ -86,6 +86,7 @@ cdef enum attr_id_t:
     ENT_TYPE
     HEAD
     SENT_START
+    SENT_END
```

Comment:
We need to place `SENT_END` at the end of the list for now -- see the comment on the `symbols.pyx`.